src/styles/theme.scss

```diff
@@ -220,7 +258,25 @@ h5 {
 }
 .section--highlight {
   color: $text-colour;
-  background-color: $grey-light-purple;
+  background-color: $white-pure;
```

Comment:
Is pure not too stark?

Reply:
I think I will change the value to `transparent`.